### PR TITLE
Handle limit errors

### PR DESCRIPTION
**Pull request by rubenmoya**
Related to: https://github.com/CartoDB/torque/issues/287

- Handle `429` status code for limit errors.
- Update node to `6.9.2`
- Update mapnik to `3.5.14`
- Add `libpango1.0-dev`, `gcc-4.9` and `g++-4.9` to travis to fix the build (needed for the new mapnik version)

### CODE REVIEW

**Comment by rubenmoya** [comment]
[This test](https://github.com/CartoDB/torque/blob/master/test/animator.js#L12) fails sometimes, I'm not sure if it's related to the `setTimeout`...

File: package.json [hunk 1]
```diff
@@ -32,9 +32,9 @@
     "browserify": "~7.0.0",
     "canvas": "~1.2.1",
     "leaflet": "1.0.3",
-    "mapnik": "cartodb/node-mapnik#1.4.15-cdb8",
```
Comment (matallo) [inline package.json]: aren't we using our fork anymore?

/cc @dgaubert

File: lib/torque/provider/windshaft.js [hunk 2]
```diff
@@ -295,14 +295,17 @@
...
+      torque.net.get( url + (extra ? "?" + extra: ''), function (response) {
         prof_fetch_time.end();
-        if (data && data.responseText) {
-          var rows = JSON.parse(data.responseText);
-          callback(self.proccessTile(rows, coord, zoom));
+        if (response && response.responseText) {
+          var body = JSON.parse(response.responseText);
+
+          response.status === 429
+            ? errorCallback(body.errors_with_context[0])
```
Comment (matallo) [inline lib/torque/provider/windshaft.js]: good for now, we'd probably want to send the tile info to draw the error tile, to be done on a separate PR

File: package.json (hunk 1 above)
Comment (dgaubert) [reply package.json]: I would try to upgrade to version `3.5.14` from upstream.
